cowamm/profitability/competitor_kpis/pancakeswap/pancakeswap_largest_pools_4232597.sql

```diff
@@ -0,0 +1,64 @@
+-- Computes the TVL for every Pancakeswap pool
+-- Then returns the top {{number_of_pools}} pools by TVL
+-- Input: blockchain, number_of_pools to return
+
+with
+-- finds the pools which have been active since 2024-10-01
+data as (
+    select
+        p.contract_address as pool_address,
+        p.call_block_time as time,
+        p.output__reserve0 as balance0,
+        p.output__reserve1 as balance1,
+        rank() over (partition by p.contract_address order by p.call_block_time desc) as latest
+    from pancakeswap_v2_{{blockchain}}.PancakePair_call_getReserves as p
```

Comment:
Is there not a better way to get this data. `_call_` tables are populated if a view function is invoked as part of a transaction, but this doesn't always fave to be the case. This was a source of inaccuracy for the initial CoW AMM performance queries. In order to get reserves we should also build a complete transfer ledger as we do for CoW AMM (unless there is already a persisted Dune view that accomplishes this)